app/models.py

```diff
@@ -24,11 +27,21 @@ def random_id():
         num = randint(1, 24)
         return num
 
-class Garden(db.Model):
-    __tablename__ = 'gardens'
+class GardenPlant(db.Model):
+    __tablename__ = 'garden_plants'
     id = db.Column(db.Integer, primary_key=True)
     plant_id = db.Column(db.Integer, db.ForeignKey('plants.id'))
+    garden_id = db.Column(db.Integer, db.ForeignKey('gardens.id'))
     plant_name = db.Column(db.String(60))
     last_watered = db.Column(db.DateTime)
     date_added = db.Column(db.DateTime, default=datetime.now())
     harvest_date = db.Column(db.DateTime)
+    
+    plant = relationship("Plant")
+
+class Garden(db.Model):
+    __tablename__ = 'gardens'
+    id = db.Column(db.Integer, primary_key=True)
+
+    plants = relationship("Plant", secondary="garden_plants")
```

Comment:
This was WAY easier than I thought it would be!  Very excited that we were able to get methods like garden.gardenplants to work... and now we can be all like... garden.gardenplants.plants[0].water_needs and get values that will help us calculate things!  Great work!